src/renderers/wikimedia-mobile.renderer.ts

```diff
@@ -130,6 +131,16 @@ export class WikimediaMobileRenderer extends MobileRenderer {
     return doc
   }
 
+  private unhideSectionsImpl(doc: DominoElement) {
+    const sections = doc.querySelectorAll('section')
+    Array.from(sections).forEach((section: DominoElement) => {
+      // Domino doesn't allow us to easily manipulate specific styles. Rather than trying to parse
+      // the style attribute and remove display: none, we just clobber the whole thing.
+      section.style = ''
```

Comment:
Just wondering if you tried `section.style.display = ''`. Your comment suggests you think this might not work, and I don't know Domino, but it seems like a strange omission if Domino is designed to implement the DOM spec in NodeJS. I only suggest trying this (if you didn't already) because it might save headaches in the future in case some other attribute other than display is added to the style element. However, of course your code has the same practical effect given the cases we're currently aware of.

Reply:
Yes unfortunately that was the first thing I tried. As you guessed, it's a Domino limitation. I read through the Domino source code to figure that out. They don't support properties of the `style` property. It's just a string.